integration_tests/tests/logger/assert_pretty_output_msg_is_string.sql

```diff
@@ -1,4 +1,4 @@
-{% if dbt_utils.pretty_log_format() is string %}
+{% if pretty_log_format() is string %}
     {# Return 0 rows for the test to pass #}
     select 1 limit 0
 {% else %}
```

Comment:
@jtcohen6 thought we'd be extra and help make ALL macros use dispatch... but will this break any references to `dbt_utils.some_macro()` in other dbt packages? for example:
- [dbt-external-tables](https://github.com/fishtown-analytics/dbt-external-tables/blob/a3c78be7bceaf76329fedc149ce8d34d734eebfb/macros/common/stage_external_sources.sql#L44)
- [dbt's integration tests ](https://github.com/fishtown-analytics/dbt/blob/28c36cc5e211f7c41bd5ca177c9253183c697097/test/integration/100_rpc_test/deps_models/main.sql#L1)

Reply:
You want to keep this as `dbt_utils.pretty_log_format()`!

If the macro call is just `pretty_log_format`, dbt will only look for a macro called `pretty_log_format` in (a) the user's own project or (b) the default (`dbt`) namespace. Macros from installed packages need to be fully qualified with their package name. With the addition of `dispatch` + the `dbt_utils_dispatch_list` var, _another_ package can interject when dbt goes to implement the fully-qualified macro `dbt_utils.pretty_log_format()`.

It's a bit tricky!

Reply:
I _think_ I understand. Perhaps I've been spared having to learn this from primarily working with adapters instead of packages. very much appreciate you coaching me through all of this, I'll revert now.

Reply:
Yup, definitely not saying it's a good thing :( FWIW `dbt-utils` is where it feels trickiest